### PR TITLE
Updated quick setup to include new instruction backend 

### PR DESCRIPTION
**Pull request by Sarahb-da**
Note: there is a lot of renaming in this, hence the large diffs

### CODE REVIEW

File: ui/src/pages/QuickSetup/LoginPage.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@use 'themes/variables' as *;
```
Comment (Sarahb-da) [inline ui/src/pages/QuickSetup/LoginPage.scss:1]: this file is just a copy of the styling code that used to be in `QuickSetup.scss` but I thought it deserved its own file

File: ui/src/pages/QuickSetup/ReviewPage.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
+@use 'themes/variables' as *;
```
Comment (Sarahb-da) [inline ui/src/pages/QuickSetup/ReviewPage.scss:1]: AGAIN: this file is just a copy of the styling code that used to be in `QuickSetup.scss` but I thought it deserved its own file

File: ui/src/pages/QuickSetup/QuickSetup.tsx [hunk 3]
```diff
@@ -17,25 +17,34 @@ import { WellKnownPartiesProvider } from '@daml/hub-react/lib';
...
+import LoginPage from './LoginPage';
 import Widget from '../../components/Widget/Widget';
+import AssignRolesPage from './AssignRolesPage';
+import ProvideServicesPage from './ProvideServicesPage';
+import CreateAccountPage from './CreateAccountPage';
+import AssignProvidersPage from './AssignProvidersPage';
 
 export enum MenuItems {
   ADD_PARTIES = 'add-parties',
+  ASSIGN_PROVIDERS = 'assign-providers',
```
Comment (alexmatson-da) [inline ui/src/pages/QuickSetup/QuickSetup.tsx:35]: Would calling this `Configure providers` instead of `Assign providers` make more sense?

**Review by alexmatson-da — APPROVED** [review]
looks good!

**Comment by alexgraham-da** [comment]
nice, lgtm!

File: ui/src/pages/QuickSetup/QuickSetup.tsx (hunk 3 above)
Comment (Sarahb-da) [reply ui/src/pages/QuickSetup/QuickSetup.tsx:35]: Oh, Will do!!

**Pull request merged**